configuration.go

```diff
@@ -109,6 +133,16 @@ func (config *Configuration) update(other *Configuration) *Configuration {
 	if other.Logger != nil {
 		config.Logger = other.Logger
 	}
+	if other.Endpoint != "" {
+		config.Logger.Printf("WARNING: the Bugsnag configuration parameter 'Endpoint' is deprecated in favor of 'Endpoints'")
+		config.Endpoint = other.Endpoint
```

Comment:
If this is set then `config.Endpoints.Sessions` should be  unset, and disable session tracking.